lookin/aiolookin/protocol.py

```diff
@@ -71,33 +109,27 @@ def connection_made(self, transport: asyncio.DatagramTransport) -> None:
 
     def datagram_received(self, data: bytes, addr: Any) -> None:
         """Process incoming state changes."""
-        # LOOKin:Updated!{device id}:{sensor id}:{event id}:{value}
-        # LOOKin:Updated!{device id}:{service name}:{value}
-        try:
-            content = data.decode()
-            if not content.startswith("LOOK.in:Updated!"):
-                return
-            _, msg = content.split("!")
-            contents = msg.split(":")
-            if len(contents) == 4:
-                self.subscriptions.notify(
-                    {
-                        "device_id": contents[0],
-                        "sensor_id": contents[1],
-                        "event_id": contents[2],
-                        "value": contents[3],
-                    }
-                )
-                return
-            self.subscriptions.notify(
-                {
-                    "device_id": contents[0],
-                    "service_name": contents[1],
-                    "value": contents[2],
-                }
-            )
-        except (KeyError, ValueError, UnicodeDecodeError):
-            pass
+        content = data.decode()
+        if not content.startswith("LOOK.in:Updated!"):
+            return
+        _, msg = content.split("!")
+        contents = msg.split(":")
+        update: dict[str, str] = {"device_id": contents[0]}
+        if len(contents) == 3:
+            # LOOK.in:Updated!{device id}:{service name}:{value}
```

Comment:
This is the actual message sent by the device

https://documenter.getpostman.com/view/11774062/SzzkddLg#eeb9f500-93cd-4f5c-8753-a628063edc10